chore: Remove OSSRH deployment configuration

esolitos opened the pull request:
Removes the OSSRH deployment configuration from the project, as it is no longer needed. The changes include:

- Removing the `ossrh` server configuration from the `.buildkite/settings-publish.xml` file.
- Removing the `ossrh-deploy-vespa` profile from the `dependency-versions/pom.xml` and `hosted-tenant-base/pom.xml` files.

esolitos committed:
chore: Remove OSSRH deployment configuration

Removes the OSSRH deployment configuration from the project, as it is no longer needed. The changes include:

- Removing the `ossrh` server configuration from the `.buildkite/settings-publish.xml` file.
- Removing the `ossrh-deploy-vespa` profile from the `dependency-versions/pom.xml` and `hosted-tenant-base/pom.xml` files.

bjorncs reviewed (dismissed):
🧹

gjoranv reviewed (commented):
The `/profiles` end tag must be removed in both of the changed poms.

esolitos committed:
feat(release): increase deployment timeout and add credentials

Increases the deployment timeout from 10 to 30 minutes.

esolitos commented:
Changes:
- I reverted the changes on the comments (damn autoformatter) 
- Removed the `</profiles>` from dependency-versions/pom.xml, but not the other pom, as it has more profiles.
